api/aggregators/documentAggregator.js

```diff
@@ -253,3 +253,26 @@ exports.createDocumentAggr = (populate, roles, sortingValue, sortField, sortDire
 
   return aggregation;
 };
+
+exports.addUpdatedInLast30daysAggr = () => {
```

Comment:
Is it possible to move this function to aggregateHelper and make it common to both project and document. Only that for project we have to add the "dateadded" field before comparing.